common/paths.cpp

```diff
@@ -51,7 +51,7 @@ void setRootPath(const QString& rootPath)
   Q_ASSERT(QDir(rootPath).exists());
   Q_ASSERT(QDir(rootPath).isAbsolute());
 
-  s_rootPath = rootPath;
+  s_rootPath = QFileInfo(rootPath).absoluteFilePath();
```

Comment:
For which case is this needed? The assert in the line above suggests rootPath is already absolute here, no?